src/settings/handlers/AbstractLocalStorageSettingsHandler.ts

```diff
@@ -108,4 +108,9 @@ export default abstract class AbstractLocalStorageSettingsHandler extends Settin
     public isSupported(): boolean {
         return localStorage !== undefined && localStorage !== null;
     }
+
+    public reset(): void {
+        AbstractLocalStorageSettingsHandler.clear();
+        AbstractLocalStorageSettingsHandler.clear();
```

Comment:
is this supposed to be calling the same thing twice?

Reply:
Derp

Reply:
Turns out Lifecycle already had a custom `clear` method it called, wired it up to the standard SettingsHandler::reset which exists now instead, turns out this was only an issue for tests so changing the T